shopping_mall/backend/ai/agent/order_graph/nodes.py

```
@@ -0,0 +1,445 @@
+    from app.models.order import Order
+
+    db = _get_db(config)
+
+    # 소유권 재검증 — defense-in-depth (list_orders에서 이미 필터했으나 재확인)
+    order = db.query(Order).filter(
+        Order.id == state["order_id"],
+        Order.user_id == state["user_id"],
+    ).first()
+    if not order:
+        logger.warning(
+            f"[order_graph] 소유권 검증 실패: user={state['user_id']} order={state['order_id']}"
```

Ruff (0.15.10)

[warning] Logging statement uses f-string

(G004)